src/main/java/uk/co/fivium/els/categories/ventilationunits/model/VentilationUnitsForm.java

```diff
@@ -0,0 +1,68 @@
+package uk.co.fivium.els.categories.ventilationunits.model;
+
+import uk.co.fivium.els.model.FieldPrompt;
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+
+public class VentilationUnitsForm {
+
+  @FieldPrompt("Supplier's name or trade mark")
+  @NotBlank(message = "Enter a supplier name or trade mark") // TODO max length?
+  private String supplierName;
+
+  @FieldPrompt("Supplier's model identifier")
+  @NotBlank(message = "Enter a supplier model identifier")
+  private String modelName;
+
+  @FieldPrompt("Energy efficiency indicator for an 'average' climate")
+  @NotBlank(message = "Select an energy efficiency class")
```

Comment:
The wording is inconsistent on the EU page (efficiency indicator/class), but I'd change the validation message to match (i.e. "select an energy efficiency indicator")